src/shared/components/paginationControls/PaginationControls.tsx

```diff
@@ -147,6 +149,13 @@ export class PaginationControls extends React.Component<IPaginationControlsProps
         }
     }
 
+    private get shouldHidePaginationControls() {
+        return (this.props.hidePaginationIfOnePage
+            && this.props.itemsPerPageOptions
+            && typeof this.props.totalItems!=="undefined"
```

Comment:
you can just use this.props.totalItems !== undefined.   I believe typeof === "undefined" is necessary only to support IE8

Reply:
I think you're right when it comes to undefined properties on an object. I think I was used to it because I'm pretty sure if its not an undefined property but a possibly undefined variable you may get a reference error, or something like that.